### PR TITLE
[DF] Move test_splitcoll_arrayview to gtests from roottest

### PR DESCRIPTION
**Pull request by eguiraud**
Complements [this PR](https://github.com/root-project/roottest/pull/464) in roottest.

### CODE REVIEW

**Comment by eguiraud** [comment]
@phsft-bot build

**Comment by eguiraud** [comment]
@phsft-bot build please?

**Comment by eguiraud** [comment]
@phsft-bot build

File: tree/dataframe/test/TwoFloatsLinkDef.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#ifdef __CINT__
```
Comment (oshadura) [inline tree/dataframe/test/TwoFloatsLinkDef.h]: Can be just replaced by `#ifdef __ROOTCLING__` :)

**Comment by oshadura** [comment]
@phsft-bot build with flags -DCTEST_TEST_EXCLUDE_NONE=On

File: tree/dataframe/test/TwoFloatsLinkDef.h (hunk 1 above)
Comment (eguiraud) [reply tree/dataframe/test/TwoFloatsLinkDef.h]: done!

**Pull request merged**